content/projects/linux/beginner/_index.md

```diff
@@ -3,6 +3,10 @@ title: Beginner Linux challenges
 ready: true
 ---
 
+### Submission guidelines
+
+For this specific project we require you to complete the required scripts in terminal, verify that your commands work and store them in a .sh file. Make sure your tasks are labeled clearly. You will be pushing this file to GitHub and submitting the github link for assessment after you complete your GitHub exercises.
```

Comment:
This should be worded a bit better. They have multiple .sh files that they need to save.  And telling them that they'll be pushing their stuff to Github will probably just confuse them.

Maybe something like this is better:

While you work through this project you will create a number of files called shell scripts, they have the extension .sh. You'll be handing those in later. In general we use a tool called Git and a platform called Github for submitting our projects, but we haven't covered that stuff yet.  etc